jni_gen/examples/notif_plugin/pubspec.yaml

```diff
@@ -0,0 +1,76 @@
+name: notif_plugin
```

Comment:
Can we call it `show_notification_plugin` or `notification_plugin` instead of `notif_plugin`? And also the folder name.

Me asking a rename again. 🙈  (When in doubt, never shorten names.)

Reply:
my_unix_stockholm_syndrome